package.json

```diff
@@ -45,7 +45,12 @@
     "@babel/plugin-proposal-class-properties": "^7.0.0-beta.43",
     "@babel/plugin-proposal-object-rest-spread": "^7.0.0-beta.43",
     "@babel/preset-env": "^7.0.0-beta.43",
+    "@babel/preset-react": "^7.0.0-beta.44",
+    "babel-core": "7.0.0-bridge.0",
```

Comment:
do we need this? there's already `@babel/core`

Reply:
We need that because babel-jest [doesn't yet support Babel 7](https://facebook.github.io/jest/docs/en/getting-started.html#additional-configuration).